package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Welcome to [LMHT].  This is an open source code repository to help Spanish speakers translate and understand HTML tags.  LMHT was originally started as a [YWebCA] project by Team Open Brace in the Summer of 2015. [Adorable] and other contributors are working towards expanding this project into a valuable learning tool for bilingual programmers.",
   "main": ".eslintrc.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "eslint --ignore js/vendor js"
```

Comment:
Two things:

1. is `--ignore js/vendor` needed, given you have an `.eslintignore` file?
2. it should be `$(npm bin)/eslint ...`